backend/test_server.py

```
@@ -99,20 +105,110 @@ def test_get_boolean_answer():
     }
     response = make_post_request(endpoint, data)
-    print(f'/get_boolean_answer Response: {response}')
-    assert 'output' in response
+    print(f'/get_boolean_answer Response: {response.json()}')
+    assert response.status_code == 200
+    assert 'output' in response.json()
 
 def make_post_request(endpoint, data):
     url = f'{BASE_URL}{endpoint}'
     headers = {'Content-Type': 'application/json'}
     response = requests.post(url, headers=headers, data=json.dumps(data))
```

Ruff (0.14.10)

Probable use of `requests` call without timeout

(S113)